Hestia/Hestia/frontend/EntryScreen/UIViewControllerLocalGlobal.cs

```diff
@@ -155,7 +155,9 @@ void SetValuesAndSegueToDevicesMainLocal()
         void SetValuesAndSegueToServerSelectGlobal()
         {
             Globals.LocalLogin = false;
-            List<HestiaServerInteractor> serverInteractors = new List<HestiaServerInteractor>();
+            Globals.Auth0Servers = new List<WebServer>();

```

Comment:
Why was this changed back? `WebServer` and `ServerInteractor` do not exist anymore.